platform/pulpcore/app/serializers/fields.py

```diff
@@ -30,3 +35,89 @@ def to_internal_value(self, data):
 
     def to_representation(self, value):
         return str(value)
+
+
+class ContentArtifactsField(serializers.DictField):
+    """
+    A serializer field for the 'artifacts' ManyToManyField on the Content model.
```

Comment:
Class attributes can be specified with [sphinx `:attribute:` directive](http://www.sphinx-doc.org/en/stable/domains.html?highlight=%3Aattr%3A#directive-py:attribute) to improve our docs.